### DISCUSSION TITLE
Amount Transfer not working

### DISCUSSION THREAD

**Discussion by a4ashishjangid** [deemru/WavesKit#18] [Q&A]
I am using deemru\WavesKit PHP sdk to transfer the amount to the wallet and it was working since 11 June but it stopped working after that.

**Comment by deemru** [comment]
![image](https://github.com/deemru/WavesKit/assets/10706633/1329c38d-c89d-446b-8b37-c6ca06aa7723)

![image](https://github.com/deemru/WavesKit/assets/10706633/19212746-8430-4619-90f1-6d05d9d92542)
